lang/src/org/partiql/lang/eval/Exceptions.kt

```diff
@@ -110,18 +110,18 @@ fun fillErrorContext(errorContext: PropertyValueMap, location: SourceLocationMet
     }
 }
 
-fun fillErrorContext(errorContext: PropertyValueMap, metaContainer: MetaContainer) {
-    val location = metaContainer.find(SourceLocationMeta.TAG) as? SourceLocationMeta
+fun fillErrorContext(errorContext: PropertyValueMap, metaContainer: com.amazon.ionelement.api.MetaContainer) {
```

Comment:
nit: don't need the `com.amazon.ionelement.api.` part before `MetaContainer` since it was imported

Reply:
Will remove them in the next commit.